src/control/lib/spdk/ctests/SConscript

```diff
@@ -0,0 +1,23 @@
+"""Build go-spdk bindings C tests"""
+import daos_build
+from os.path import join
+
+def scons():
+    """Execute build"""
+    Import('senv', 'prereqs')
+
+    unit_env = senv.Clone()
+    prereqs.require(unit_env, 'pmdk', 'spdk', 'hwloc', 'cmocka')
+
+    # equivalent to -I flags for spdk/lib/nvme and spdk/include for ut mocking
+    spdk_src = unit_env.subst("$SPDK_SRC")
+    unit_env.AppendUnique(CPPPATH=[join(spdk_src, "lib", "nvme"),
```

Comment:
Are we assuming SPDK_SRC is available?  If it’s a test, should we make this conditional?

Reply:
what's the best way to do that?